### ISSUE TITLE
Gradient w.r.t input is NoneType

### ISSUE THREAD

**Issue by WenFuLee** [uber-research/UPSNet#24]
I wanted to observe the gradient w.r.t input (that is, the input image here).
So, I tried to print `data['data'].grad` after `loss.backward()` below.
https://github.com/uber-research/UPSNet/blob/master/upsnet/upsnet_end2end_train.py#L216
But I got an error saying that `data['data'].grad` is NoneType.
Is there anything I misunderstand or missed? Any idea would be appreciated. Thanks.

**Comment by WenFuLee** [comment]
Hi, does anyone know how I can solve this issue? I am still stuck with this. Thanks.
